library/src/main/java/com/yausername/youtubedl_android/YoutubeDL.kt

```diff
@@ -254,20 +443,17 @@ object YoutubeDL {
         }
     }
 
+    internal val scope = CoroutineScope(SupervisorJob())
```

Comment:
Be cautious to use coroutine in this library, or I'll just suggest avoiding coroutines. Because we don't know, and I believe we don't want to learn how coroutines behave in a Java world.

Reply:
We have to use coroutines for Ktor; I tested the library download dependencies/plugins download in the Java example app and works perfect, no issues found. So I don't know if other solution can exist

Reply:
I've deleted this value and added runBlocking in IO threads so the actual app thread isn't blocked, If you have any other alternative you tell me; by the moment gonna mark this review as resolved.